tools/delete-fixtures.js

```diff
@@ -0,0 +1,32 @@
+const { resolve } = require('path');
+const { rm, readdir } = require('fs/promises');
+
+// Get fully resolved paths of all sub-directories in a directory
+async function getDirectories(dir) {
+  return (await readdir(dir, { withFileTypes: true }))
+    .filter(dirent => dirent.isDirectory())
+    .map(dirent => resolve(dir, dirent.name))
+}
+
+// Recursively get all sub-directories including the root directory
+async function getDirectoriesRecursivePlusRoot(dir) {
+  const subDirs = await getDirectories(dir)
+
+  const list = await Promise.all(subDirs.map((subDir) => getDirectoriesRecursivePlusRoot(subDir)))
+  return [dir, ...list.reduce((a, f) => a.concat(f), [])]
```

Comment:
nit - would `list.flat()` work instead of the `reduce`?

Reply:
yep changed